DirectProgramming/DPC++/GraphTraversal/hidden-markov-models/README.md

```diff
@@ -0,0 +1,90 @@
+#`DPC++ Hidden Markov Model` Sample
+HMM (Hidden Markov Model) is the statistical model in which the system is represented by a Markov process with the unobservable or so called "hidden" states. 
+States can be represented as nodes of directed graph. The directed edges of this graph are possible transitions beetween nodes defined with some probabilities.
```

Comment:
You have several formatting errors, please fix them. if you need assistance hit me up via IM